### PR TITLE
Add Storybook decorators

### PR DESCRIPTION
**Pull request by JosephusPaye**
This adds a number of convenience decorators for stories

- Font (`conf/storybook/decorators/font.tsx`): Sets the font family used in NUsight.
- Fullscreen: (`src/client/components/storybook/fullscreen.tsx`): Reset body padding and set width and height to fill viewport
- Container: (`src/client/components/storybook/container.tsx`): Create a decorator with a max-width container with optional padding

Storybook: https://nusight-pr-235.herokuapp.com/storybook/

### CODE REVIEW

File: src/client/components/select/stories/select.stories.tsx [hunk 1]
```diff
@@ -12,21 +13,16 @@ const actions = {
...
 
-const Container = ({ children }: { children: any }) => {
-  return <div style={{ maxWidth: '320px', fontFamily: 'Arial, sans-serif' }}>
-    {children}
-  </div>
-}
+const container = createContainerDecorator({ maxWidth: '350px' })
 
 storiesOf('components.select', module)
+  .addDecorator(container)
```
Comment (BrendanAnnable) [inline src/client/components/select/stories/select.stories.tsx]: I think this one is too specific to be generic, can we just inline the decorator here? e.g.

```ts
.addDecorator((story: RenderFunction) => {
  return <div style={{ maxWidth: 350 }}>{story()}</div>
})
```

File: src/client/components/storybook/fullscreen.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { RenderFunction } from '@storybook/react'
+import * as React from 'react'
+
+import * as style from './fullscreen.css'
+
+export function fullscreen(story: RenderFunction) {
+  return <div className={style.fullscreen}>
+    <style>{`body { margin: 0; padding: 0; }`}</style>
```
Comment (BrendanAnnable) [inline src/client/components/storybook/fullscreen.tsx]: Clever! I'm sure there is a better way to achieve this haha, but this is probably fine for now.

Can you add a comment why this is needed?

File: src/client/components/select/stories/select.stories.tsx (hunk 1 above)
Comment (JosephusPaye) [reply src/client/components/select/stories/select.stories.tsx]: Yeah, makes sense.

Comment (JosephusPaye) [reply src/client/components/select/stories/select.stories.tsx]: Done!

File: src/client/components/storybook/fullscreen.tsx (hunk 2 above)
Comment (JosephusPaye) [reply src/client/components/storybook/fullscreen.tsx]: Done!

**Pull request merged**